linux_os/guide/services/nfs_and_rpc/disabling_nfs/disabling_nfs_services/service_rpcbind_disabled/rule.yml

```diff
@@ -24,6 +24,8 @@ identifiers:
 references:
     cis: 2.2.7
 
+platform: machine
```

Comment:
Containers have services too. Reverting this change.

Reply:
This is some mistake, you are actually adding the machine-only specification.

Reply:
@matejak The original PR preferred marking the group as machine-only and then removed from this rule. See `linux_os/guide/services/nfs_and_rpc/disabling_nfs/disabling_nfs_services/group.yml`. The rule was previously marked as machine only already before the original PR.